test-apps/display-test-app/ios/backend.config.js

```diff
@@ -43,33 +43,40 @@ function getConfig(env) {
     devtool: "source-map",
     resolve: { mainFields: ["main", "module"] },
     module: {
+      // don't parse @bentley/imodeljs-native/NativeLibrary.js,
+      // we don't need to pull in the Native here as it gets loaded by the runtime
+      // via (process as any)._linkedBinding("iModelJsNative")
+      noParse: [/NativeLibrary.js$/],
       rules: [
         {
           test: /AzCopyFileHandler\.js/g,
-          use: 'null-loader'
+          use: "null-loader",
         },
         {
           test: /itwin\+electron-authorization/g,
-          use: 'null-loader'
+          use: "null-loader",
         },
         {
           test: /ElectronBackend\.js/g,
-          use: 'null-loader'
-        }
-      ]
+          use: "null-loader",
+        },
+      ],
     },
     stats: {
-      warnings: false
+      warnings: false,
     },
     externals: {
-      "electron": "electron",
-      "bufferutil": "bufferutil",
-      "utf-8-validate": "utf-8-validate"
+      electron: "electron",
+      bufferutil: "bufferutil",
+      "utf-8-validate": "utf-8-validate",
     },
     plugins: [
       new plugins.CopyAppAssetsPlugin("./assets/"),
       new plugins.CopyBentleyStaticResourcesPlugin(["assets"]),
-      new webpack.DefinePlugin({ "global.GENTLY": false, "process.version": "'v10.9.0'" })
+      new webpack.DefinePlugin({
```

Comment:
how closely does this match what is in the MobileSDK samples?

Reply:
Mobile isnt using the Copy plugins, i'm not certain yet that's right. Will update configs once we're able to verify everything on the latest mobile sdk that works

Reply:
okay, sounds good.

Reply:
do we want to hold this pr for this, or just come back and update configs as needed later?

Reply:
no need to hold it. We can straighten that out later, I agree with you that there is probably stuff in the mobile samples that isn't necessary either.